src/Microsoft.DocAsCode.Plugins/IHostService.cs

```diff
@@ -3,22 +3,27 @@
 
 namespace Microsoft.DocAsCode.Plugins
 {
+    using System;
     using System.Collections.Immutable;
 
     public interface IHostService
     {
         ImmutableList<TreeItemRestructure> TableOfContentRestructions { get; set; }
 
+        [Obsolete("use GroupInfo")]
         /// <summary>
         /// current version's name, String.Empty for default version
         /// </summary>
         string VersionName { get; }
 
+        [Obsolete("use GroupInfo")]
```

Comment:
should we place the attribute below the comment section...

Reply:
yes...should